.env

```diff
@@ -32,8 +32,12 @@ TEST_LOG_REQUESTS=false
 # Default admin credentials - ALSO OVERRIDE IN PRODUCTION
 ADMIN_EMAIL=ni@aefeup.pt
 ADMIN_PASSWORD=n1j0bs_ftw.12345
+
 #CORS allowed origin - OVERRIDE IN PRODUCTION
 ACCESS_CONTROL_ALLOW_ORIGIN= 
```

Comment:
Remove this var and just use the new one (remember to edit the config/env.js)

Reply:
@BrunoRosendo We'll have to update .env in our server